### PR TITLE
Hasna

### PR DESCRIPTION
**Pull request by HasnaTrk**
can you check

### CODE REVIEW

File: src/Group1/g1.java [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 public class g1 {
     public static void main(String[] args) {
-        System.out.println("test16");
+        System.out.println("test105");
         System.out.println("test222");
         System.out.println("test333");
         System.out.println("I am adding something");
```
Comment (Filizbagdat) [inline src/Group1/g1.java]: can you  delete line 8 please

**Review by Filizbagdat — CHANGES_REQUESTED** [review]

File: src/Group1/g1.java (hunk 1 above)
Comment (HasnaTrk) [reply src/Group1/g1.java]: I deleted

**Review by Filizbagdat — APPROVED** [review]

File: src/Group1/g1.java (hunk 1 above)
Comment (Filizbagdat) [reply src/Group1/g1.java]: thank you

Comment (Filizbagdat) [reply src/Group1/g1.java]: image
Test105 is true please update

Comment (HasnaTrk) [reply src/Group1/g1.java]: Ok I deleted test105 and wrote Test105

Comment (Filizbagdat) [reply src/Group1/g1.java]: thank you

**Pull request merged**